index.html

```diff
@@ -90,136 +137,116 @@ <h2 class="multi-post">Multi-Post stories</h2>
+              <li><img src="./Images/Counter.png" alt="Dot-2 image"></li>
+              <li class="year">2015</li>
+            </ul>
+          </div>
+        </div>
 
-      <h2 class="tonic">Tonic</h2>
-      <div class="list-1">
-        <ul class="list-1">
-          <li class="canopy">CANOPY</li>
-          <li class="dot"><img src="./Images/Counter.png" alt="Dot-1 image"></li>
-          <li class="job-title">Back End Dev</li>
-          <li><img src="./Images/Counter.png" alt="Dot-2 image"></li>
-          <li class="year">2015</li>
+        <p class="short-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <ul class="buttons">
+          <li><button class="html-btn">html</button></li>
+          <li><button class="css-btn">css</button></li>
+          <li><button class="js-btn">javascript</button></li>
         </ul>
+        <div class="project-btn"><button class="pro-btn">See Project</button></div>
       </div>
-
-      <p class="short-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-      <ul class="buttons">
-        <li><button class="html-btn">html</button></li>
-        <li><button class="css-btn">css</button></li>
-        <li><button class="js-btn">javascript</button></li>
-      </ul>
-      <div class="project-btn"><button class="pro-btn">See Project</button></div>
     </div>
   </section>
 
   <section>
-    <div class="pages">
-
-      <div><img src="./Images/snap2.png" alt="body image"></div>
+    <div class="grid-container2">
+      <div class="head">
+        <h1 class="about-heading">About Myself</h1>
+        <p class="my-intro">Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</p>
+        <p class="connect-to">LET'S CONNECT</p>
+        <ul class="icons-2">
+          <li><a href="#"><img src="./Images/github.png" alt="icon"></a></li>
+          <li><a href="https://www.linkedin.com/in/khawanin-ali-zada-93777b21a/"><img src="./Images/linkedin.png"
+                alt="linkedin icon"></a></li>
+          <li><a href="#"><img src="./Images/vector.png" alt="icon"></a></li>
+          <li><a href="https://twitter.com/AliKhawanin?t=WjCWTIUHm2vvE734-wThJQ&s=03"><img src="./Images/twitter.png"
+                alt="twitter icon"></a></li>
+          <li><a href="mailto:khawaninalizada21@gmail.com"><img src="./Images/gmail.png" alt="Mail icon"></a></li>
+        </ul>
+        <div><button class="resume">Get my resume</button></div>
+      </div>
+      <div class="head-img2">
+        <ul class="languages">
+          <li class="lang">
+            <h2>Languages</h2>
+          </li>
+          <li><img src="./Images/Union1.png" alt="downward icon"></li>
+        </ul>
 
-      <h2 class="multi-post">Multi-Post stories</h2>
```

Comment:
Kindly modify the _about me_ section to match the design on Figma. Check the comment above for more reasons you should stick to a given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187010136-e77d3f22-4845-48d0-8487-81a401dc8107.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187010120-e29265f5-53eb-4979-96e3-b303495701fe.png)